sites/public/pages/listings.tsx

```diff
@@ -183,8 +183,8 @@ const ListingsPage = () => {
   // eslint-disable-next-line @typescript-eslint/unbound-method
   const { handleSubmit, register, errors } = useForm()
   const onSubmit = (data: ListingFilterState) => {
-    if (data[FrontendListingFilterStateKeys.includeNulls] === false) {
-      delete data[FrontendListingFilterStateKeys.includeNulls]
+    if (data?.[FrontendListingFilterStateKeys.includeNulls] === false) {
+      delete data?.[FrontendListingFilterStateKeys.includeNulls]
```

Comment:
We changed it for the filters, but I noticed it wasn't working correctly when I ran this change.  Adding it resolved several errors.

Reply:
What were the errors? Since I thought that accessing it like an array should return something falsey if the key didn't exist. If there was an error where the data object was null/undefined, that seems like a new problem. Also, I'm unclear why this would be impacted with this PR?

Reply:
Once you exit the eligibility questionnaire and hit reset, you get this error:
![image](https://user-images.githubusercontent.com/82653098/135290562-45387737-ec82-4f96-8f07-3291712fa12a.png)
Enabling the optional feature allowed the reset to work properly from my understanding as the data (form information) wouldn't exist in that case.  Previously we were only testing it from the model.

Reply:
merged and resolved issue